src/components/cocktail.js

```diff
@@ -6,22 +6,31 @@ import { GET_A_RECOMMENDATION } from '../constants.js';
 import AppDispatcher from '../core/dispatcher.js';
 import cocktailStore from '../stores/cocktail_store.js';
 import spinnerStore from '../stores/spinner_store.js';
+import { Link } from 'react-router-dom'
 
 export default class Cocktail extends React.Component {
 
   constructor(props) {
     super(props);
+    console.log(props);
     this.history = createBrowserHistory();
     this.loc = React.createRef()
     this.state = {
-      loc: ''
+      loc: '',
+      cocktail: props.cocktail
     }
   }
 
   updateQ = (e) => {
     this.setState({loc: this.loc.current.value})
   }
 
+  // shouldComponentUpdate = (nextProps, nextState) => {
+  //   console.log(this.props, this.state);
+  //   console.log(nextProps, nextState);
+  //   return true;
+  // }
+
```

Comment:
Is this for a future iteration or something we'll use later?

Reply:
it was the old code that may not be needed anymore but I didn't want to delete it out of the file yet. I guess I can

Reply:
removed